### PR TITLE
refactor: migrate apy from new api endpoint [WEB-355]

### PR DESCRIPTION
**Pull request by xgambitox**
- Refactor to use APY from new API endpoint, merging data to old schema.

### CODE REVIEW

File: app/containers/Vaults/saga.js [hunk 1]
```diff
@@ -61,14 +62,65 @@ const injectEthVaults = (vaults) => {
...
+      ...vault,
+      apy: {
+        ...vault.apy,
+        recommended: newApy.net_apy,
+        data: {
+          ...vault.apy.data,
+          grossApy: newApy.gross_apr,
+          netApy: newApy.net_apy,
+          ...(newApy.composite && {
+            totalApy: newApy.composite.pool_apy
```
Comment (turtlemoji) [inline app/containers/Vaults/saga.js:86]: we should really validate this with all the type of vaults

Comment (xgambitox) [reply app/containers/Vaults/saga.js]: changed to use gross_apr @dudesahn

**Comment by turtlemoji** [comment]
Use 
image
for backscratcher

**Comment by yearn-integrations** [comment]
## Important changes detected  (payment related code)
- Found address [0x9d409a0A012CFbA9B15F6D4B36Ac57A46966Ab9a](https://etherscan.io/address/0x9d409a0A012CFbA9B15F6D4B36Ac57A46966Ab9a) in files app/containers/Vaults/saga.js. This address is from vault `yvBOOST`  
- Found address [0xFBEB78a723b8087fD2ea7Ef1afEc93d35E8Bed42](https://etherscan.io/address/0xFBEB78a723b8087fD2ea7Ef1afEc93d35E8Bed42) in files app/containers/Vaults/saga.js. This address is from vault `yvUNI`

**Review by turtlemoji — APPROVED** [review]
approve code, must revalidate data though

**Comment by yearn-integrations** [comment]
## Important changes detected  (payment related code)
- Found address [0x9d409a0A012CFbA9B15F6D4B36Ac57A46966Ab9a](https://etherscan.io/address/0x9d409a0A012CFbA9B15F6D4B36Ac57A46966Ab9a) in files app/containers/Vaults/saga.js. This address is from vault `yvBOOST`  
- Found address [0xFBEB78a723b8087fD2ea7Ef1afEc93d35E8Bed42](https://etherscan.io/address/0xFBEB78a723b8087fD2ea7Ef1afEc93d35E8Bed42) in files app/containers/Vaults/saga.js. This address is from vault `yvUNI`

**Review by x48-crypto — APPROVED** [review]

**Pull request merged**